Containerfile.redis

```diff
@@ -1,33 +1,17 @@
 # Use Red Hat Universal Base Image (UBI) 9 as the base image
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-# Install necessary dependencies (gcc, make, etc.) for building Redis from source
-RUN microdnf install -y gcc make wget tar gzip && \
-    microdnf clean all
+# Install necessary dependencies for downloading and installing Redis
+RUN microdnf install -y wget yum-utils logrotate systemd && \
+    wget https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/redis-7.0.12-1.module_el9+499+353c08ce.x86_64.rpm && \
```

Comment:
better.  wish we could just dnf install the rpm from the repo directly but it doesn't seem to show up for some reason.

Reply:
(even when i tried using a centos streams base image and dnf installing into that).  maybe @williamcaban has an idea